README.md

```diff
@@ -1,19 +1,25 @@
-# uobtheatre-api
+#uobtheatre-api
 
 ![Gitub actions](https://github.com/BristolSTA/uobtheatre-api/workflows/Python%20package/badge.svg?branch=main)
 
 API for uobtheatre. Check out the project's [documentation](http://BristolSTA.github.io/uobtheatre-api/).
 
 # Prerequisites
 
```

Comment:
can we have the correct version of python in here, and then pip/pipenv install commands